code/game/objects/items/weapons/storage/backpack_vr.dm

```diff
@@ -12,7 +12,7 @@
 
 	mob_can_equip(var/mob/living/carbon/human/H, slot, disable_warning = 0)
 		if(..())
-			if(istype(H) && istype(H.tail_style, taurtype))
+			if(istype(H) && H.is_taur())
```

Comment:
Is it by design that the changes in this file seem to revert a previous PR made recently?

Reply:
Hmmm. I may have messed that file up. I was attempting to use that new helper proc in here as an example of how it could work, but if it reverted another PR I'll undo this. Oops.